src/lint/collect-spelling-diagnostics.ts

```diff
@@ -34,6 +34,23 @@ let matchers = [
     pattern: /[ \t]+\n/gu,
     message: 'Trailing spaces are not allowed',
   },
+  {
+    pattern: /(?<=\n)\n\n+/gu,
```

Comment:
```suggestion
    pattern: /(?<=(^|[^\n])\n\n)\n+/gu,
```

This does two things:

1. Points to the second empty line, not the first.
1. Only raises a single linting error, even for very long runs of empty lines.

You should add tests confirming these things.

Reply:
Done.